tidb.go

```diff
@@ -243,9 +243,24 @@ func NewStore(uri string) (kv.Storage, error) {
 
 var queryStmtTable = []string{"explain", "select", "show", "execute", "describe", "desc"}
 
+func trimSQL(sql string) string {
```

Comment:
I test tidb-server and it is ok now.

Database changed
mysql> select /*comment*/ 1;
+------+
| 1    |
+------+
|    1 |
+------+
1 row in set (0.00 sec)

mysql> /*comment*/ select 1;
+------+
| 1    |
+------+
|    1 |
+------+
1 row in set (0.00 sec)

mysql>